lib/src/klio/transforms/io.py

```diff
@@ -398,3 +406,92 @@ def _get_file_pattern(self, file_pattern, location):
             file_pattern = location
 
         return file_pattern
+
+
+# note: fast avro is default for py3 on beam
+class _KlioFastAvroSink(beam_avroio._FastAvroSink):
```

Comment:
if my struggles with BQ have been any indication, we should be careful about inheriting internal beam classes, since they seem to have no qualms with sudden breaking changes.  Could this be implemented as a custom `PTransform` that bundles a `beam.Map` with the standard `WriteToAvro`, similarly to what I did with [KlioReadFromBigQuery](https://github.com/spotify/klio/blob/master/lib/src/klio/transforms/io.py#L269)?

Don't think it's a blocker for this PR, just something we should be aware of